vlog4j-owlapi/src/main/java/org/semanticweb/vlog4j/owlapi/OwlAxiomToRulesConverter.java

```diff
@@ -219,14 +221,19 @@ void startAxiomConversion() {
 	 * @param superClass
 	 */
 	void addSubClassAxiom(final OWLClassExpression subClass, final OWLClassExpression superClass) {
-		this.startAxiomConversion();
-
-		final ClassToRuleHeadConverter headConverter = new ClassToRuleHeadConverter(this.frontierVariable, this);
-		superClass.accept(headConverter);
-		final ClassToRuleBodyConverter bodyConverter = new ClassToRuleBodyConverter(this.frontierVariable,
-				headConverter.body, headConverter.head, this);
-		bodyConverter.handleDisjunction(subClass, this.frontierVariable);
-		this.addRule(bodyConverter);
+		if (subClass instanceof OWLObjectOneOf) {
+			final OWLObjectOneOf subClassInstaceOf = (OWLObjectOneOf) subClass;
```

Comment:
`subClassInstanceOf` is misspelled.

Reply:
I actually renamed it to `subClassObjectOneOf`, as it is more fitting.